src/libraries/System.Private.CoreLib/src/System/CodeDom/Compiler/IndentedTextWriter.cs

```diff
@@ -56,6 +56,17 @@ public int Indent
         /// <inheritdoc/>
         public override Task FlushAsync() => _writer.FlushAsync();
 
+        /// <summary>
+        /// Clears all buffers for this IndentedTextWriter asynchronously and causes any buffered data to be
```

Comment:
This could either be the cref, or just `writer`.

```suggestion
        /// Clears all buffers for this <see cref="IndentedTextWriter"/> asynchronously and causes any buffered data to be
```